src/FlanCommand.ts

```diff
@@ -33,9 +33,10 @@ export default abstract class FlanCommand extends Command {
       saveDir: ".flan/local",
       repoDir: ".flan/repo",
       database: {
-        host: "localhost",
+        host: "",
```

Comment:
we can keep this as localhost since we override with PGHOST

Reply:
Makes sense. I'll put a default port as well.